### ISSUE TITLE
spark:hive is does not work

### ISSUE THREAD

**Issue by ppalaga** [apache/camel-quarkus#1956]
Blocked by https://issues.apache.org/jira/browse/CAMEL-15696

**Comment by ppalaga** [comment]
Won't fix, because we plan to remove the existing JVM-only Spark extension https://github.com/apache/camel-quarkus/issues/3466
